### PR TITLE
Add Xinxin Zhu to authors.yml

### PR DESCRIPTION
**Pull request by starsz**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_data/authors.yml [hunk 1]
```diff
@@ -113,6 +113,18 @@ brian-demers:
...
 
+xinxin-zhu:
+  full_name: Xinxin Zhu
+  display_name: Xinxin Zhu
+  avatar: avatar-xinxinzhu.jpg
+  email: xinxinzhu@api7.ai
+  github: https://github.com/starsz
+  bio:
+    Xinxin Zhu is a System Engineer at API7.AI and a committer for the Apache APISIX project. He is an open source lover and
+    a golang fans. Now, he is working on control plane of Apache APISIX such as writing code, reviewing code and answering questions
```
Comment (yzeng25) [inline _source/_data/authors.yml:124]: fans --> fan

File: _source/_data/authors.yml [hunk 2]
```diff
@@ -113,6 +113,18 @@ brian-demers:
...
+    in the community. You can find him on Twitter at <a href="https://twitter.com/starszcan">@starszcan</a>,
+    he is a very funny persion. :)
```
Comment (yzeng25) [inline _source/_data/authors.yml:126]: he is a fun guy.

Funny  and fun have slightly difference. Fun means "有趣", funny means "滑稽".

**Review by yzeng25 — CHANGES_REQUESTED** [review]

File: _source/_data/authors.yml (hunk 1 above)
Comment (starsz) [reply _source/_data/authors.yml:125]: Fix. Use developer instead.

File: _source/_data/authors.yml (hunk 2 above)
Comment (starsz) [reply _source/_data/authors.yml:127]: Fixed.

**Review by yzeng25 — APPROVED** [review]

File: _source/_data/authors.yml [hunk 3]
```diff
@@ -113,6 +113,19 @@ brian-demers:
...
+  display_name: Xinxin Zhu
+  avatar: avatar-xinxinzhu.jpg
+  email: xinxinzhu@api7.ai
+  twitter: https://twitter.com/starszcan
+  github: https://github.com/starsz
+  bio:
+    Xinxin Zhu is a System Engineer at API7.AI and a committer for the Apache APISIX project. He is an open source enthusiast and
+    a Golang developer. Now, he is working on control plane of Apache APISIX such as writing code, reviewing code and answering questions
+    in the community. You can find him on Twitter at <a href="https://twitter.com/starszcan">@starszcan</a>,
+    he is a fun guy. :)
```
Comment (bdemers) [inline _source/_data/authors.yml:127]:
```suggestion
    Xinxin Zhu is a System Engineer at API7.AI and a committer for the Apache APISIX project. He is an open-source enthusiast and
    a Golang developer. Now, he is working on the control plane of Apache APISIX writing code, reviewing code, and answering questions
    in the community. Xinxin Zhu is a fun guy, you can find him on Twitter at <a href="https://twitter.com/starszcan">@starszcan</a> 🙂
```

**Review by bdemers — APPROVED** [review]
Thanks!! This is great!

**Comment by starsz** [comment]
> Thanks!! This is great!

Much appreciated!

**Review by bdemers — APPROVED** [review]

**Pull request merged**